lib/rex/script/meterpreter.rb

```diff
@@ -4,17 +4,24 @@ module Rex
 module Script
 class Meterpreter < Base
 
-begin
-  include Msf::Post::Windows::Priv
-  include Msf::Post::Windows::Eventlog
-  include Msf::Post::Common
-  include Msf::Post::Windows::Registry
-  include Msf::Post::File
-  include Msf::Post::Windows::Services
-  include Msf::Post::Windows::Accounts
-rescue ::LoadError
-end
+  begin
+    include Msf::Post::Windows::Priv
+    include Msf::Post::Windows::Eventlog
+    include Msf::Post::Common
+    include Msf::Post::Windows::Registry
+    include Msf::Post::File
+    include Msf::Post::Windows::Services
+    include Msf::Post::Windows::Accounts
+  rescue ::LoadError
+  end
 
+  def initialize(client, path)
+    # The mixins for `Msf::Post::*` now assume a single info argument is present,
```

Comment:
For a paper trail, it was the PR to keep track of meterpreter compatibility information
https://github.com/rapid7/metasploit-framework/pull/15079/files#diff-a44df7f377f2f240b034ca610b54a20a03c658309c37da7e0a1660fb4913628a